lib/vsc/utils/mail.py

```diff
@@ -105,6 +130,10 @@ def _send(self,
                 raise
             except smtplib.SMTPDataError as err:
                 raise
+
+            if self.auth:
```

Comment:
what does this do? (and in particular, why is it only needed / allowed if auth is used?)